spec/features/staff/users_can_view_a_fund_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.feature "Users can view a fund" do
+  before do
+    authenticate!(user: user)
+  end
+
+  let(:organisation) { create(:organisation) }
+  let(:organisation_2) { create(:organisation) }
+  let!(:viewable_fund) { create(:fund, organisation: organisation) }
+  let!(:forbidden_fund) { create(:fund, organisation: organisation_2) }
+  let(:user) { create(:user, organisations: [organisation]) }
+
+  context "when the user is not logged in" do
+    it "redirects the user to the root path" do
+      page.set_rack_session(userinfo: nil)
+      visit fund_path(viewable_fund)
+      expect(current_path).to eq(root_path)
+    end
+  end
+
+  context "when the fund belongs to the user's organisation" do
+    scenario "the user can view the fund" do
+      visit fund_path(viewable_fund)
```

Comment:
Again, I think we should have at least one test that tests this end-to-end and ensures we have navigation in place.